requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+flake8
+pytest
+suds-jurko
+requests
```

Comment:
The basic dependencies are 'pip installable' but `geoalchemy-odm2` in requirements-dev.txt is not.